ACPs/77-reinventing-subnets/README.md

```diff
@@ -94,116 +240,38 @@ type ConvertSubnetTx struct {
     Message warp.Message `json:"message"`
 }
 ```
 
-The `Message` field must be an `AddressedCall` with the payload:
-
-```text
-+--------------+----------+-----------+
-|      codecID :   uint16 |   2 bytes |
-+--------------+----------+-----------+
-|       typeID :   uint32 |   4 bytes |
-+--------------+----------+-----------+
-|     subnetID : [32]byte |  32 bytes |
-+--------------+----------+-----------+
-|       nodeID : [32]byte |  32 bytes |
-+--------------+----------+-----------+
-|       weight :   uint64 |   8 bytes |
-+--------------+----------+-----------+
-| blsPublicKey : [48]byte |  48 bytes |
-+--------------+----------+-----------+
-|       expiry :   uint64 |   8 bytes |
-+--------------+----------+-----------+
-                          | 134 bytes |
-                          +-----------+
-```
+The `validationID` of validators added via `RegisterSubnetValidatorTx` is defined as the SHA256 hash of the `Payload` of the `AddressedCall` in `Message`.
 
-- `codecID` is the codec version used to serialize the payload and is hardcoded to `0x0000`
-- `typeID` is the payload type identifier and is `0x00000001` for this transaction
-- `subnetID`, `nodeID`, `weight`, and `blsPublicKey` are for the Subnet Validator being added
-- `expiry` is the time at which this message becomes invalid. As of a P-Chain timestamp `>= expiry`, this Avalanche Warp Message can no longer be used to add the `nodeID` to the validator set of `subnetID`
+When a `RegisterSubnetValidatorTx` is accepted on the P-Chain, the Subnet Validator is added to the Subnet's validator set. A `minNonce` field corresponding to the `validationID` will be stored on addition to the validator set (initially set to `0`). This field will be used when validating the `SetSubnetValidatorWeightTx` defined below.
 
-    `validationID` of validators added via `RegisterSubnetValidatorTx` is defined as the SHA256 hash of the `Payload` of the `AddressedCall`. This SHA256 hash will be used for replay protection. Used `validationID`s will be stored on the P-Chain. If a `RegisterSubnetValidatorTx`'s `validationID` has already been used, the transaction will be considered invalid. To prevent storing an unbounded number of `validationID`s, the `expiry` is required to be no more than 48 hours in the future of the time the transaction is issued on the P-Chain. Any `validationIDs` corresponding to an expired timestamp can be flushed from the P-Chain's state.
+This `validationID` will be used for replay protection. Used `validationID`s will be stored on the P-Chain. If a `RegisterSubnetValidatorTx`'s `validationID` has already been used, the transaction will be considered invalid. To prevent storing an unbounded number of `validationID`s, the `expiry` of the `RegisterSubnetValidatorMessage` is required to be no more than 48 hours in the future of the time the transaction is issued on the P-Chain. Any `validationIDs` corresponding to an expired timestamp can be flushed from the P-Chain's state.
```

Comment:
48h is likely too much. Currently avalanchego has it set to 24h, it might need to be even smaller.

Reply:
Totally defer to whatever you think is safe from a P-Chain perspective. 

UX wise, I think as low as 30 minutes is sufficient.